packages/search-in-workspace/src/browser/search-in-workspace-result-tree-widget.tsx

```diff
@@ -573,7 +573,8 @@ export class SearchInWorkspaceResultTreeWidget extends TreeWidget {
         const icon = node.icon;
         return <div className='result'>
             <div className='result-head'>
-                <div className={`result-head-info noWrapInfo noselect ${node.selected ? 'selected' : ''}`}>
+                <div className={`result-head-info noWrapInfo noselect ${node.selected ? 'selected' : ''}`}
+                    title={new URI(node.fileUri).path.toString()}>
```

Comment:
How about `labelProvider.getLongName(new URI(node.fileUri))`?

Reply:
`labelProvider.getLongName(new URI(node.fileUri))` returns relative path, but when the workspace is multi root workspace, we should return absolute path

Reply:
ok for now

Generally, we should use `getLongName` and adjust its implementation properly depending on the current setup in order to have consistent way of representing paths everywhere.

Reply:
OK, I will keep this advice in mind and make sure they have consistent way of representing paths.